frontend/packages/network-attachment-definition-plugin/integration-tests/tests/models/nadDetail.ts

```diff
@@ -0,0 +1,49 @@
+import { browser } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { clickHorizontalTab } from '@console/internal-integration-tests/views/horizontal-nav.view';
+import { createYAMLLink, isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { isLoaded as yamlPageIsLoaded } from '@console/internal-integration-tests/views/yaml.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import * as NADListView from '../../views/nad.list.view';
+import { NADForm } from './nadForm';
+
+export class NADDetail {
+  readonly name: string;
+
+  readonly namespace: string;
+
+  constructor(instance) {
+    this.name = instance.name;
+    this.namespace = instance.namespace || 'all-namespaces';
+  }
+
+  async navigateToListView() {
+    const nadsListURL = (namespace) =>
+      `${appHost}/k8s/ns/${namespace}/k8s.cni.cncf.io~v1~NetworkAttachmentDefinition/`;
```

Comment:
It's probably not used ATM, but just for the sake od correctness, this won't work for `all-namespaces`, because for all-namespaces, the `/ns/` is not included in the URL. See https://github.com/openshift/console/blob/0ff7e4a27cafb46d054abd0b41bc1d82ee72ae7f/frontend/packages/kubevirt-plugin/integration-tests/tests/models/detailView.ts#L46 for the fixed code

Reply:
Done!